<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>580</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>建模协会制～</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>参赛时间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_month">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>61</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_year">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>61</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_datemask">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入年月日：如2016-09-18</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>321</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入密码</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示查询</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>男</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>复选框</string>
    </property>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>105</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>105</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>国家级</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>105</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>891</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>获得比赛选项</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>导入文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_rstquery_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>苹方</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>392</width>
      <height>214</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAsd">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionA"/>
    <addaction name="actionB"/>
   </widget>
   <addaction name="menuAsd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_showpic"/>
   <addaction name="action_showtab"/>
  </widget>
  <action name="actionA">
   <property name="text">
    <string>导入文件</string>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>导出文件</string>
   </property>
  </action>
  <action name="action_showpic">
   <property name="text">
    <string>绘图展示</string>
   </property>
   <property name="toolTip">
    <string>绘图展示</string>
   </property>
  </action>
  <action name="action_showtab">
   <property name="text">
    <string>表格展示</string>
   </property>
   <property name="toolTip">
    <string>表格展示</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
